backend/.eslintrc.json

```diff
@@ -31,7 +31,7 @@
 
     // Uncomment this if you have verbatimModuleSyntax enabled and importing
     // .ts files with a .js extension gives lint errors.
-    // "import/no-unresolved": "off",
+    "import/no-unresolved": "off",
```

Comment:
I couldn't get firebase to import properly, and people online say that we either need a custom import solution to fix this. This was the easiest fix I found, but it's a temporary patch.

Reply:
This should be fine. If you look in the comments above we should be able to use this eslint feature